app.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics-controller");
+
+const app = express();
+app.use(express.json());
```

Comment:
are we using this line yet? take it out until you need to use it. Think about what it's actually doing and which endpoint you'll actually need it

Reply:
I guess I need this back when I do a post request?